### ISSUE TITLE
Add test for boundary chunks issue

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-io#56]
Could we add a test for whether boundary chunks are written and read correctly? For example, validating that the values at the edge of the image are correct?!

**Comment by K-Meech** [comment]
Yes - will do! It's a bit of a difficult one, as this issue only occurs with images above a certain size (where it writes more levels / reads from previously written levels). Also, the chunks it writes aren't corrupt, the values are just wrong - so it's a bit hard to check for. The easiest would probably be to generate a random image (of a big enough size), then write it to n5 and ome-zarr with the same settings. Then I can check if the lowest resolution level values are the same between the two.

**Comment by K-Meech** [comment]
@tischi - what's the best way to check if two RandomAccessibleInterval have exactly the same pixel values?

**Comment by tischi** [comment]
This is how you can copy one RAI into another:

```
public static < T extends Type< T > > void copy(
			final RandomAccessible< T > source,
			final IterableInterval< T > target )
	{
		// create a cursor that automatically localizes itself on every move
		Cursor< T > targetCursor = target.localizingCursor();
		RandomAccess< T > sourceRandomAccess = source.randomAccess();

		while ( targetCursor.hasNext() )
		{
			targetCursor.fwd();
			targetCursor.get().set( sourceRandomAccess.setPositionAndGet( targetCursor ) );
		}
	}
```

Small modifications (some if statement in the while loop) of this would be probably allow you to compare the pixel values.

**Comment by constantinpape** [comment]
Solved by #57
